display: 1
num_red: 31
num_green: 27
num_blue: 27
num_black: 20
